docs/example.rst

```diff
@@ -1,50 +1,64 @@
-**************************
-Example: Sea state contour
-**************************
+**************************************
+Quick start example: Sea state contour
+**************************************
 
-Let's start this user guide with a simple example.
+Let’s start this user guide with a simple example, which can be copied easily line by line or downloaded from the
+examples_ section of the repository. Based on a dataset, the long-term joint distribution of sea states is estimated,

```

Comment:
Based on measurement data...

Reply:
I think dataset is fine. "dataset" can also be used consistently, because we also use hindcast data later. One could argue hindcast data are not "measurement data".